.github/renovate.json5

```diff
@@ -12,16 +12,11 @@
   "separateMinorPatch": true,
   "packageRules": [
     {
-      "matchPackageNames": [
-        "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator",
-        "io.opentelemetry.proto:opentelemetry-proto",
-        "io.opentelemetry.semconv:opentelemetry-semconv"
-      ],
       // Renovate's default behavior is only to update from unstable -> unstable if it's for the
       // major.minor.patch, under the assumption that you would want to update to the stable version
       // of that release instead of the unstable version for a future release
-      // (TODO remove once the artifacts above release stable versions)
-      "ignoreUnstable": false
+      "matchCurrentVersion": "/^.*-alpha$/",
```

Comment:
see https://github.com/renovatebot/renovate/discussions/13849#discussioncomment-2057657

Reply:
extracted to standalone PR: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/10761

Reply:
done